run-tests.sh

```diff
@@ -83,6 +83,10 @@ elif [[ "${1:-}" = "--exec-test" ]]; then
     export KERNEL_SELFTESTS="/run/input/bpf"
   fi
 
+  if [[ -f "/run/input/bpf/bpf_testmod/bpf_testmod.ko" ]]; then
+    insmod "/run/input/bpf/bpf_testmod/bpf_testmod.ko"
```

Comment:
Shipped in selftests.tgz as of [7744b00ee32e31f6c43d79911d27d704b45de83a](https://github.com/cilium/ci-kernels/commit/7744b00ee32e31f6c43d79911d27d704b45de83a)